ui/selectors/selectors.js

```diff
@@ -1080,6 +1080,31 @@ export const getAllConnectedAccounts = createDeepEqualSelector(
   },
 );
 
+export const getMemoizedCurrentChainId = createDeepEqualSelector(
+  getCurrentChainId,
+  (chainId) => chainId,
+);
+
+export const getMemoizedTxId = createDeepEqualSelector(
+  (state) => state.appState.txId,
+  (txId) => txId,
+);
```

Comment:
Hey @FrederikBolding : are deep selectors required for single values like ids above.

Reply:
I believe so yeah, since it is used to compare the input state value and not the `txId` itself.